org.opent2t.sample.lamp.superpopular/com.hue.bulb/js/colour.js

```diff
@@ -0,0 +1,359 @@
+"use strict";
+
+const ChangeTolerance = 0.0001;
+const MaxHue = 360.0;
+const MaxColour = 255;
+
+/*
+ * Method to see if the given XY value is within the reach of the lamps.
+ *
+ * @param xyColour is a two-element object representing a point containing the X,Y value
+ * @param colourGamut   the color gamut of the light device.
+ * @return true if within reach, false otherwise.
+ */
+function isPointInLampsReach(xyColour, colourGamut){
```

Comment:
Resolved below -- OCF uses the British variant so then we stick with that -- it would be nice to answer this once and for all and have a quick comment say in the manifest pointing to the OCF spec 

---
In reply to: [94470752](https://github.com/openT2T/translators/pull/143#discussion_r94470752) [](ancestors = 94470752)